src/components/Modal.vue

```diff
@@ -0,0 +1,76 @@
+<template>
+  <div
+    id="modal-fadeInScale-fs"
+    :class="{
+      'modal is-active': isActive,
```

Comment:
try this:

```suggestion
      'modal is-active animated zoomIn': isActive,
```

Reply:
So actually I think it's better if we do that to the modal content since doing that to the main modal div animates the whole blurred background around the content. Looks better when we do it that way.

Reply:
Done ^ in c525c21